### ISSUE TITLE
is there a way to Tag teams instead of username ?

### ISSUE THREAD

**Issue by khooj002** [ashwanthkumar/gocd-slack-build-notifier#97]
What I am trying to do is to tag team instead of a slack user 

for example, 
`owner` field in Slack accept username such as "ashwanthkumar".
However, instead of tagging "@ashwanthkumar", I wanna tag the team "ashwanthkumar" belongs to. like <!subteam^IDMYS7814|developer-team>

**Comment by ashwanthkumar** [comment]
@kicketster I haven't personally tried it but can you try adding the team name like `@developer-team` and see if it works?

**Comment by khooj002** [comment]
Yeah I tried it already, it doesn’t work. If you are tagging team in Slack, it has to be in the format from the above example.

**Comment by jdavisp3** [comment]
I'm interested in working on this. Any guidance on how you'd like to see this structured @ashwanthkumar ?

**Comment by ashwanthkumar** [comment]
@jdavisp3 I would assume it would be a new field in the configuration may be?
